### ISSUE TITLE
FR: Filename in collection items

### ISSUE THREAD

**Issue by IOIO72** [DanWebb/jdown#15]
I'd like to use the filenames of the markdown files of collections as the document part of the URL.

It would be nice to get some meta data, related to the markdown files - especially for collections.

This is a FAQ example:

```json
{
"faq": [
        {
                "category": "general",
                "question": "How can I share content?",
                "tags": [
                        "all",
                        "app"
                ],
                "contents": "<p>Something</p>\n"
        },
        {
                "category": "general",
                "question": "Another question...",
                "tags": [
                        "all",
                        "app"
                ],
                "contents": "<p>Something else</p>\n"
        }
]
}
```

The new collection items could be extended by a new meta object:

```json
{
"faq": [
        {
                "category": "general",
                "question": "How can I share content?",
                "tags": [
                        "all",
                        "app"
                ],
                "contents": "<p>Something</p>\n",
                "@meta": {
                    "filename": "how-share.md",
                    "changed": "1557327938829"
                }
        },
        {
                "category": "general",
                "question": "Another question...",
                "tags": [
                        "all",
                        "app"
                ],
                "contents": "<p>Something else</p>\n",
                "@meta": {
                    "filename": "another-question.md",
                    "changed": "1557327976828"
                }
        }
]
}
```

**Comment by IOIO72** [comment]
Oops.... I realized, this duplicates the closed issue #13  .... sorry

**Comment by DanWebb** [comment]
Hey @IOIO72 yeah check out the fileInfo config option https://github.com/DanWebb/jdown#fileinfo I'll close this issue for now but let me know if you have any trouble implementing!
